cloudvolume/frontends/graphene.py

```diff
@@ -36,7 +37,7 @@ def to_unix_time(timestamp):
   if isinstance(timestamp, datetime): # NB. do not change to elif
     timestamp = datetime.timestamp(timestamp)
 
-  if not isinstance(timestamp, int) and timestamp is not None:
+  if not isinstance(timestamp, float) and timestamp is not None:
```

Comment:
This should read `isinstance(timestamp, (int, float))` and should return `int(timestamp)` I think. Using a UNIX timestamp directly is supported and will usually be regarded as an int.